### PR TITLE
Added option to switch to UDP for controls

### PR DESCRIPTION
**Pull request by wouterheerwegh**
- [x] Added the ROS param `UDP_control` to switch between UDP and TCP
- [x] Added docs explaining when to use it

### CODE REVIEW

File: docs/getting-started.md
Comment (SijmenHuizenga) [inline docs/getting-started.md:39]: I believe the binary should work placed anywhere. What kind of black screen with some buttons did you encounter that made you conclude that the binary has to be in the user folder?

File: docs/getting-started.md
Comment (SijmenHuizenga) [inline docs/getting-started.md:100]: Can you include the `local-setup.md` file?

File: ros/src/fsds_ros_bridge/launch/fsds_ros_bridge.launch
Comment (SijmenHuizenga) [inline ros/src/fsds_ros_bridge/launch/fsds_ros_bridge.launch:10]: Would you like to add this argument in the arguments table in the `ros-bridge.md` docs file?

File: ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp
Comment (SijmenHuizenga) [inline ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp:190]: ❤️

**Review by SijmenHuizenga — CHANGES_REQUESTED** [review]
Code looks great! Let's get all documentation sorted, after that we are ready to merge

File: docs/getting-started.md
Comment (wouterheerwegh) [reply docs/getting-started.md:39]: I was referring to issue #224, might be fixed by now

**Review by SijmenHuizenga — COMMENTED** [review]
Thank you so much for writing this documentation! You really got my mind crunching about how to best explain multi-computer setups and the possible problems that can occur. 

You did a great job explaining what people should do. I think we can do an even better job explaining the forces behind the recommendations in an effort to help people make informed decisions about their setup.

At the moment, the `Local setup` tries to achieve two things:

1. Explain the two different setups: one where the ROS bridge is running on the simulator computer and one where the ROS bridge runs on the autonomous system computer.
2. Explain how to get around the WSL bug

To make 1 and 2 super clear, and not scare people who are using both C++ and Python, I suggest we split the two. In this mindset I have re-structured the documentation. I've also moved it into `getting-started-with-ros.md` as there is more then enough space in there for these docs.

I'm looking forward to hear your opinion on this!

**Comment by wouterheerwegh** [comment]
Looks a lot better indeed, I'll add the changes to the pull request. You should be able to push to the pull request though, the 'allow edits by maintainers' option is checked

**Review by SijmenHuizenga — APPROVED** [review]

**Pull request merged**
